Makefile

```
@@ -221,6 +345,17 @@ wait-for-prod-health: ## Wait for the prod container to reach a healthy state.
 		fi; \
 	done
 
+.PHONY: visual-direct e2e-direct all clean
```

checkmake (0.2.2)

[warning] Missing required phony target "test"

(minphony)